server/zoom/meeting.go

```diff
@@ -3,39 +3,24 @@
 
 package zoom
 
-// MeetingType as defined at https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate
-type MeetingType int
-
-const (
-	// MeetingTypeInstant meeting
-	MeetingTypeInstant MeetingType = 1
-	// MeetingTypeScheduled meeting
-	MeetingTypeScheduled MeetingType = 2
-	// MeetingTypeRecurringWithNoFixedTime meeting
-	MeetingTypeRecurringWithNoFixedTime MeetingType = 3
-	// MeetingTypeRecurringWithFixedTime meeting
-	MeetingTypeRecurringWithFixedTime MeetingType = 8
-)
-
 // Meeting is defined at https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meeting
 type Meeting struct {
-	UUID              string      `json:"uuid"`
-	ID                int         `json:"id"`
-	HostID            string      `json:"host_id"`
-	Topic             string      `json:"topic"`
-	Type              MeetingType `json:"type"`
-	Status            string      `json:"status"`
-	StartTime         string      `json:"start_time"`
-	Duration          int         `json:"duration"`
-	Timezone          string      `json:"timezone"`
-	CreatedAt         string      `json:"created_at"`
-	Agenda            string      `json:"agenda"`
-	JoinURL           string      `json:"join_url"`
-	StartURL          string      `json:"start_url"`
```

Comment:
Probably nothing important, but i noticed you "lost" this attribute (`StartURL`)